internal/store/bundle.go

```diff
@@ -111,15 +112,30 @@ func (b *bundleStore) List() ([]reference.Reference, error) {
 }
 
 // Remove removes a bundle from the bundle store.
-func (b *bundleStore) Remove(ref reference.Reference) error {
+func (b *bundleStore) Remove(ref reference.Reference, force bool) error {
 	if id, ok := ref.(ID); ok {
-		if len(b.refsMap[id]) == 0 {
+		refs := b.refsMap[id]
+		if len(refs) == 0 {
 			return fmt.Errorf("no such image %q", reference.FamiliarString(ref))
-		} else if len(b.refsMap[id]) > 1 {
-			return fmt.Errorf("unable to delete %q - App is referenced in multiple repositories", reference.FamiliarString(ref))
+		} else if len(refs) > 1 {
+			if force {
+				toDelete := append([]reference.Reference{}, refs...)
+				for _, r := range toDelete {
+					if err := b.doRemove(r); err != nil {
```

Comment:
As the `force` flag has been specified here, maybe we should try removing everything then fail at the end?